samza-kv/src/main/java/org/apache/samza/storage/kv/StoreBackedReadWriteTable.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.samza.storage.kv;
+
+import java.util.List;
+
+import org.apache.samza.table.ReadWriteTable;
+
+
+/**
+ * A store backed readable and writable table
+ *
+ * @param <K> the type of the key in this table
+ * @param <V> the type of the value in this table
+ */
+public class StoreBackedReadWriteTable<K, V> extends StoreBackedReadableTable<K, V>
+    implements ReadWriteTable<K, V> {
+
+  /**
+   * Constructs an instance of {@link StoreBackedReadWriteTable}
+   * @param kvStore the backing store
+   */
+  public StoreBackedReadWriteTable(KeyValueStore kvStore) {
```

Comment:
nit: Consider to name it LocalStoreBackedReadWriteTable? Or KeyValueStoreBackedReadWriteTable?

Reply:
Fixed